cloudbuild/python-scripts/utils/utils.py

```diff
@@ -0,0 +1,249 @@
+"""Utilities for creation of BQ table."""
+
+import datetime
+import os
+import random
+import re
+import string
+import avro
+import avro.datafile
+import avro.io
+from google.cloud import bigquery
+
+
+def is_perfect_hour(datetime_obj):
+    """Returns True if the datetime object is a perfect hour, False otherwise."""
+    return (
+        datetime_obj.minute == 0
+        and datetime_obj.second == 0
+        and datetime_obj.microsecond == 0
+    )
+
+
+def generate_string():
+    return ''.join(random.choices(string.ascii_letters, k=random.randint(8, 10)))
+
+
+def generate_long():
+    return random.choice(range(0, 10000000))
```

Comment:
nit: we can also do this instead `random.randint(0, 10000000)`

Reply:
Fixed.

Reply:
Still seeing `return random.choice(range(0, 10000000)`